src/openzaak/components/documenten/widgets.py

```diff
@@ -3,5 +3,7 @@
 
 class PrivateFileWidget(_PrivateFileWidget):
     def get_display_value(self, value):
```

Comment:
```suggestion
from typing import Optional
from django.db.models.files import FieldFile

    def get_display_value(self, value: Optional[FieldFile]):
```

I _think_ it's a `FieldFile` object, you'd have to double check to get the right type hints :)